### DISCUSSION TITLE
Stream Runtime event logs into websockets

### DISCUSSION THREAD

**Discussion by vasudwivedi** [dagster-io/dagster#18828] [Q&A]
Hello, 
So, i am executing a dagster job in python using flask over it, and i want to push the event logs of the job into websockets so that i can view the status in the UI, just like we see in dagit UI but i dont know how capture and push these logs at runtime into the WebSocket. Can anyone help me out or can suggest me a different approach.

**Comment by ei-grad** [comment]
Why not use to GraphQL API as Dagster Web UI does? Here is its TypeScript code - https://github.com/dagster-io/dagster/blob/9285bc9/js_modules/dagster-ui/packages/ui-core/src/runs/LogsProvider.tsx#L289.
